src/test/resources/warehouse/s3Credentials-update.json

```diff
@@ -0,0 +1 @@
+{"s3Credentials":{"region":"region","accessKey":"accessKey","secretKey":"secretKey"}}
```

Comment:
this file duplicates s3Credentials-create.json - is that intentional

Reply:
got rid of duplication by having a modified `secretKey` in update request.